pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -127,7 +129,7 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pa *pav1alpha1.PodAutosc
 		mode = nv1alpha1.SKSOperationModeProxy
 	}
 
-	sks, err = c.ReconcileSKS(ctx, pa, mode)
+	sks, err = c.ReconcileSKS(ctx, pa, mode, decider.Status.NumActivators)
```

Comment:
Do we need to guard against a 0 and default to MinActivators in that case too? We can reach this point before the scaler has even run, right?

Reply:
I originally considered using "all" if "0" received in the SKS.
But this works just as well.
Done.